build.gradle

```
@@ -55,14 +55,14 @@ ext {
     MC_VERSION = '1.18.2'
     MCP_VERSION = '20220404.173914'
     SPI_VERSION = '4.0.15-4.x'
-    ACCESSTRANSFORMERS_VERSION = '8.0.4'
-    COREMODS_VERSION = '5.0.1'
-    EVENTBUS_VERSION = '5.0.3'
+    ACCESSTRANSFORMERS_VERSION = '8.1.4'
```

```suggestion
    ACCESSTRANSFORMERS_VERSION = '8.1.1'
```
8.1.2 removes ANTLR. To use that version or higher, you need to make sure the same ANTLR version it used to use is still bundled with Forge to ensure no intentional breaking changes.